sips/sip-026/sip-026-sbtc_peg.md

```diff
@@ -0,0 +1,171 @@
+# SIP-028: sBTC Bootstrap
+
+- **SIP Number:** 028
+- **Title:** A Decentralized and Programmable Asset Backed 1:1 with BTC 
```

Comment:
This SIP is supposed to be describing the _criteria for signers_ for such a system, not the system itself.  The title should reflect this.